board/ti/am335x/mux.c

```diff
@@ -399,7 +399,7 @@ void enable_board_pin_mux(void)
 		configure_module_pin_mux(gpio0_7_pin_mux);
 		configure_module_pin_mux(rgmii1_pin_mux);
 		configure_module_pin_mux(mmc0_pin_mux_sk_evm);
-	} else if (board_is_bone_lt()) {
+	} else if (board_is_oresat() || board_is_bone_lt()) {
```

Comment:
I asked about the pin mux in a meeting a few months ago because of this. I probably didn't do a good job of articulating what I was after. I wonder if our custom board files override this. The way we were using u-boot before I think it was defaulting to beaglebone black. What do you recommend? We should probably do some testing.

Reply:
Here's some questions to answer that will help:
- At what point in the boot process is this function run?
- At what point in the boot process does pinmux setup usually happen?
- Why is some but not all pinmux setup taking place here?
- Why can't these just be configured where all the other pinmux setup happens?

Reply:
U-boot is definitely defaulting to the am335x_boneblack.dtb on the oresat boards if their EEPROM still have BNLT00C0 stored in them. I'll do some digging to see when that dtb is replaced by the custom one and how it affects the mux. 

This reminds me of a question I had since I started working with Zephyr and device trees. Should we continue building these custom dts files or refactor them as overlays?